src/main/java/medicaldoctor/controllers/AdminDashboardController.java

```diff
@@ -0,0 +1,21 @@
+package medicaldoctor.controllers;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.fxml.Initializable;
+
+public class AdminDashboardController implements Initializable, ParentController{
+    ControllerManager cm;
+    
+    @Override
+    public void initialize(URL location, ResourceBundle resources) {
+        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
```

Comment:
we don't want this exception

Reply:
also I don't know what this `initialize` is specifically, but i know from the Navbar that a no-param method works.

Reply:
autogenerated I'll remove it next revision.

Reply:
yes
but put a more descriptive commit message than "revision"